c10/core/impl/LocalDispatchKeySet.h

```diff
@@ -24,67 +25,45 @@
 namespace c10 {
 namespace impl {
 
-// POD version of LocalDispatchKeySet.  Declared here just so that
-// we can put it in the guards.
-// This struct encapsulates special handling for TLS initialization
-// in set_included()/included() API so that they reflect the truth.
-// If you want to create PODLocalDispatchKeySet with non-zero state,
-// use set_included() instead of default constructor.
-struct C10_API PODLocalDispatchKeySet {
-  uint64_t included_;
-  uint64_t excluded_;
+// Helper class to handle conversion between the TLS zero initialized
+// convention and the "normal" in memory layout.
+class C10_API LocalDispatchKeySetWrapper {
```

Comment:
@swolchok I guess, making a class here isn't a pessimization? I was worried about the class members being forced to put into memory but I tried a few simple and gcc was able to eliminate the class entirely.

Reply:
Yeah, I checked that it's identical.

Reply:
as long as all the member functions are inlineable, you should be OK.